platform/index.md

```diff
@@ -22,4 +81,9 @@ redirects:
 
 [![ServicePulse article index](/menu/servicepulse-logo.png)](/servicepulse)
 
-[![ServiceControl article index](/menu/servicecontrol-logo.png)](/servicecontrol)
\ No newline at end of file
+[![ServiceControl article index](/menu/servicecontrol-logo.png)](/servicecontrol)
+
+To see demos showing how those tools are used and to understand the principles behind the platform even better, watch the videos:
+
+* [Deep dive into Service Matrix and Service Insight](http://fast.wistia.net/embed/iframe/4348umnahj?videoFoam=true&autoPlay=true), 
```

Comment:
i dont think we should call out videos here. the links to other products in the main article should be enough